packages/blockchain-info-components/index.d.ts

```diff
@@ -277,3 +277,15 @@ export const CheckBoxInput: FunctionComponent<{
   onClick?: () => void
   onChange?: () => void
 }>
+export const SelectInput: FunctionComponent<{
+  disabled?: boolean,
+  elements?: Array<any>,
+  grouped?: boolean,
+  label?: string | any,
```

Comment:
is it ok to use `any` as type option? shouldn't we use `unknown` perhaps? [Here's the PR](https://github.com/Microsoft/TypeScript/pull/24439) for the v3 of typescript which includes this new safe typing counterpart.

Reply:
yep, I think is better use `unknown`. thank you charly